english/tunnel/tunnel.md

```diff
@@ -5,3 +5,5 @@ hear your pursuer, only your breath echoing around you.
 [Look for a light.] (blind-search/blind-search.md)
 
 [Continue down the tunnel.] (continue/continue.md)
+
+[Stop and meditate on the situation](stop/stop.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.